pyanaconda/installclass.py

```diff
@@ -58,6 +58,10 @@ class BaseInstallClass(object):
     # Blivet uses by default.
     defaultFS = None
 
+    # The default partitioning scheme to use for autopartitioning.
+    # If None, we will use DEFAULT_AUTOPART_TYPE.
```

Comment:
Still better to have a constant for this -- for other installclasses, addons,...